### PR TITLE
April 30, 2024 release notes

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_help/release_notes.md [hunk 1]
```diff
@@ -54,6 +54,82 @@ guide_featured_list:
 > <br>
 > For more information on any of the updates listed in this section, reach out to your account manager or [open a support ticket]({{site.baseurl}}/help/support/). You can also check out [our SDK Changelogs]({{site.baseurl}}/developer_guide/platform_integration_guides/sdk_changelogs/) to see more information on our monthly SDK releases, updates, and improvements.
 
+## April 30, 2024 release
+
+### Permissions to create or update promotion code lists
+
+As of April 2024, users will need the “Access Campaigns, Canvases, Cards, Segments, Media Library” permission to create or update promotion code lists.
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes.md:61]: Is there a doc update or link that users can be directed to?

File: _docs/_help/release_notes.md [hunk 2]
```diff
@@ -54,6 +54,82 @@ guide_featured_list:
...
+
+### New Braze partnerships
+
+#### Friendbuy - Loyalty
+
+Leverage the integration between Braze and [Friendbuy]({{site.baseurl}}/partners/message_orchestration/channel_extensions/loyalty/friendbuy/) to expand your email and SMS capabilities while effortlessly automating your referral and loyalty program communications. Braze will generate customer profiles for all the opted-in phone numbers collected via Friendbuy.
+
+### NiftyImages - Dynamic Content
+
+The Braze and [NiftyImages](https://niftyimages.com) partnership allows you to create dynamic and personalized images for your email campaigns by mapping your existing Braze personalization tags to your NiftyImages URLs.
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes.md:109]:
```suggestion
The Braze and [NiftyImages]({{site.baseurl}}/partners/message_personalization/dynamic_content/niftyimages/) partnership allows you to create dynamic and personalized images for your email campaigns by mapping your existing Braze personalization tags to your NiftyImages URLs.
```

Comment (rachel-feinberg) [reply _docs/_help/release_notes.md:109]: Linking to the article in release notes, and linking to the external partner site in the article.

Comment (lydia-xie) [reply _docs/_help/release_notes.md:109]: Oops - thanks for catching this! Thought I updated it 😵‍💫

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Left a few comments

**Pull request merged**